Seq.Input.CertificateCheck/CertificateValidityCheck.cs

```diff
@@ -33,7 +33,7 @@ public async Task<CertificateCheckResult> CheckNow(HttpClientWrapper httpClientW
             }
             catch (Exception exception)
             {
-                diagnosticLog.Error(exception, "Something went wrong while checking certificate");
+                diagnosticLog.Error(exception, $"Something went wrong while checking certificate for URL {_targetUrl}");
```

Comment:
Since these are structured logs the target URL should be captured using a named placeholder, e.g.:

```csharp
diagnosticLog.Error(exception, "Something went wrong while checking certificate for {TargetUrl}", _targetUrl);
```

Reply:
Of course, stupid of me. Thanks for noting. Fixed.